wp-content/civi-extensions/goonjcustom/Civi/InstitutionCollectionCampService.php

```
@@ -115,26 +115,38 @@ public static function updateInstitutionDispatchDetails(string $op, string $obje
   /**
    *
    */
-  public static function updateCampStatusOnOutcomeFilled(string $op, string $objectName, int $objectId, &$objectRef) {
+  public static function updateCampStatusOnOutcomeAndACKFilled(string $op, string $objectName, int $objectId, &$objectRef) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$op'. (undefined)

(UnusedFormalParameter)